pkg/users/account-create-dialog.js

```diff
@@ -88,6 +89,14 @@ function AccountCreateBody({ state, errors, change }) {
                     </FlexItem>
                 </Flex>
             </FormGroup>
+            <FormGroup label={_("UID")}
```

Comment:
Can we spell this out as "User ID"? Then it's also easier to translate. I think we can afford the space, "full name" and "authentication" are also rather long.